app/models/spotlight/ability.rb

```diff
@@ -26,7 +26,8 @@ def initialize(user)
         Spotlight::Resource,
         Spotlight::Page,
         Spotlight::Contact,
-        Spotlight::CustomField
+        Spotlight::CustomField,
+        Translation
```

Comment:
FWIW, there is an ability spec if you want to throw in an expectation about curators being able to access translations.  We haven't been super consistent about adding things there, so don't consider this a blocker.

Reply:
I'll look into adding that.

Reply:
Added in 2059a7bd626815c13f5af3e8faf576b43d4777a2